src/vss_tools/vspec/units_quantities.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2023 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+from pathlib import Path
+from vss_tools.vspec.model import VSSUnit, VSSQuantity, ModelValidationException
+from pydantic import ValidationError
+import yaml
+from vss_tools import log
+
+
+class UnitQuantityRedefinitionException(Exception):
+    pass
+
+
+class MalformedDictException(Exception):
+    pass
+
+
+def load_units_or_quantities(
+    files: list[Path], class_type: type[VSSUnit | VSSQuantity]
+) -> dict[str, VSSUnit | VSSQuantity]:
+    data = {}
+    for file in files:
+        content = yaml.safe_load(file.read_text())
+        if not content:
+            log.warning(f"{file}, empty")
+            continue
+        log.info(
+            f"Loaded '{class_type.__name__}', file={file.absolute()}, elements={len(content)}"
```

Comment:
Should this possibly be on debug level instead, gives quite a lot of output for standard vss catalog

(based on our approach so far that you on Info-level should not get too many lines of output)

Reply:
Hm, for units and quantities I would like to show it in info, for vspec e.g. in the VSS, it is a lot of output, that's true. Will change that to maybe only show the toplevel vspec in the info and all subsequent loads in the debug